activesupport/lib/active_support/multibyte/chars.rb

```diff
@@ -148,8 +149,8 @@ def capitalize
 
       # Capitalizes the first letter of every word, when possible.
       #
-      #   "ÉL QUE SE ENTERÓ".mb_chars.titleize    # => "Él Que Se Enteró"
-      #   "日本語".mb_chars.titleize                 # => "日本語"
+      #   "ÉL QUE SE ENTERÓ".mb_chars.titleize.to_s    # => "Él Que Se Enteró"
+      #   "日本語".mb_chars.titleize.to_s               # => "日本語"
```

Comment:
without `to_s` here, the return would be an `ActiveSupport::Multibyte::Chars` instance